lighthouse-core/gather/driver.js

```diff
@@ -433,12 +433,14 @@ class Driver {
    *    See https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query.
    */
   queryPermissionState(name) {
+    if (['geolocation', 'notifications', 'midi', 'push'].indexOf(name) === -1) {
```

Comment:
Gotcha. I think leaving off `JSON.stringify` is ok since we'll be the ones writing the majority of audits.